lib/Common/CommonDefines.h

```diff
@@ -183,6 +183,12 @@
 #endif
 #endif
 
+#if ENABLE_NATIVE_CODEGEN
+#ifdef _WIN32
```

Comment:
_WIN32 contains _WIN64 (https://msdn.microsoft.com/en-us/library/b0084kay.aspx):

```
_WIN32 Defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, x86, or x64. Otherwise, undefined.

_WIN64 Defined as 1 when the compilation target is 64-bit ARM or x64. Otherwise, undefined. 
```